crates/biome_css_parser/tests/css_test_suite/error/scss/at-rule/media-interpolated-modifier-tail.scss

```
@@ -0,0 +1,4 @@
+$query: "and (color)";
+
+@media only screen #{$query} {}
```

Stylelint (17.14.1)

[error] Empty block (block-no-empty)

(block-no-empty)